### PR TITLE
Feature#18 flight reservation list

### PR DESCRIPTION
**Pull request by hancel-avila**
# Description
What did you implemented/Why did you implemented this: 
 - **What? 1**
 - We added a temporary flights screen. This is a preview screen that shows the list of user's current flights.
 - The list will show de flight origin and destination, number, date and number of passengers.
 -  **Why? 1**
 - The user will be able to review the flights registered.
 
 - **What? 2**
 - We disabled the buttons that allows user to move on to the next step when data is incomplete.
 - **Why? 2**
 - If the user hasn't complete the form or introduced valid data, he won't be able to move on to the next step.
 
  
  ## Screenshots 
image
image

### CODE REVIEW

**Review by yovanha — DISMISSED** [review]
Well done!

File: FlightReservation/src/components/flights/FlightDestiny.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <View style={style.flights_row}>
+        <Text style={style.flights_small_text_light}>Serbia</Text>
+        <Text style={style.flights_small_text_light}>Netherlands</Text>
+      </View>
+      <View style={style.separator_thin} />
+      <View style={style.flights_row}>
+        <Text style={style.flights_small_text_bold}>September 2, 2020</Text>
+        <Text style={style.flights_small_text_bold}>2 passengers</Text>
+      </View>
+      <View style={style.separator_thick} />
```
Comment (DemisRincon) [inline FlightReservation/src/components/flights/FlightDestiny.js:23]: Code climate is marking this as repeated code, please refactorize

File: FlightReservation/src/screens/FlightReservation.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+const FlightReservation = ({navigation}) => {
+  return (
+    <ScrollView>
+      <View>
+        <View style={style.textFieldView}>
+          <ArrowBack navigation={navigation} />
+          <Text style={style.title}>
+            <Span text="flights" />
+          </Text>
+        </View>
```
Comment (DemisRincon) [inline FlightReservation/src/screens/FlightReservation.js:22]: You are repeating this in another part of your code please refactorize

File: FlightReservation/src/screens/LogIn.js [hunk 3]
```diff
@@ -44,24 +45,23 @@ const [authError, setAuthError] = useState(false);
...
               </View>
             </View>
             <View>
               <View style={style.buttons_container} />
-              <Pressable
-                onPress={formProps.handleSubmit}>
+              <Pressable onPress={formProps.handleSubmit}>
                 <View style={style.btn}>
                   <Text style={style.button_text}>
                     <Span text="login" />
```
Comment (DemisRincon) [inline FlightReservation/src/screens/LogIn.js:67]: this pressable can be implemented as you ButtonForm

**Review by DemisRincon — CHANGES_REQUESTED** [review]
please fix the recomendations

File: FlightReservation/src/screens/FlightReservation.js (hunk 2 above)
Comment (hancel-avila) [reply FlightReservation/src/screens/FlightReservation.js]: Done. I created a new reusable component to replace this.

File: FlightReservation/src/screens/LogIn.js (hunk 3 above)
Comment (hancel-avila) [reply FlightReservation/src/screens/LogIn.js]: Done.
We applied the ButtonForm to habilitate/inhabilitate the button.

**Review by DemisRincon — APPROVED** [review]

**Review by cortezroberto — APPROVED** [review]

**Review by yovanha — APPROVED** [review]

**Pull request merged**